### DISCUSSION TITLE
Favoriten aus Karte entfernen

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#503] [Q&A] [answered]
Wie kann ich einen Favoriten aus der Karte entfernen das er nicht mehr sichtbar ist aber nicht im Favoritenordner gelöscht wird ?

**Comment by devemux86** [comment, marked as answer]
@1maxi2 

Open the favorites screen and use the menu of the corresponding group to change the visibility.

Öffnen Sie den Favoritenbildschirm und verwenden Sie das Menü der entsprechenden Gruppe, um die Sichtbarkeit zu ändern.

<img src="https://github.com/devemux86/cruiser/assets/3484020/64700215-0067-4fb1-b6fd-98591e3ec507" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/5fc6c37a-9436-4ba9-962c-7afd942f6c5f" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/7d0c1874-2ec5-49ea-9fe8-02b1754fd104" width="33%"/>

**Reply by 1maxi2** [reply]
Bei mir schaut das anders aus. Kann das sein da ich noch die 3.0.42 habe ?

<img src="https://github.com/devemux86/cruiser/assets/124886589/a9c29649-553f-4269-86a5-246f9aeabef2" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/124886589/fc8389c2-0d7a-4f1d-aa3b-93952057df2b" width="50%"/>

**Reply by devemux86** [reply]
@1maxi2 

Use the menu of the **groups** in the 1st tab (not the 2nd tab).

Verwenden Sie das Menü der **Gruppen** auf der 1. Registerkarte (nicht auf der 2. Registerkarte).

**Reply by devemux86** [reply]
@1maxi2 

The visibility and the icons / colors of the favorites can be changed in their **groups**.

Die Sichtbarkeit und die Symbole / Farben der Favoriten können in deren **Gruppen** geändert werden.

**Reply by 1maxi2** [reply]
Ganz begriffen hab ich das System noch nicht 🤔 Aber ich denke noch. Vielleicht kommt die Eingebung noch.
Am einfachsten wäre wenn man wie Bild einen Menüpunkt in den einzelnen Favoriten hinzufügen w
![Screenshot_20240421_154946_Cruiser](https://github.com/devemux86/cruiser/assets/124886589/fb6c281e-6909-4f97-8b38-e3aa89a333dc)
ürde " In Karte anzeigen mit Haken oder nicht ohne Haken "
Aber vielleicht denke ich zu kompliziert !

**Reply by devemux86** [reply]
@1maxi2 

Favorites cannot be made visible individually, only the entire group.
This is why this menu is available in the groups on the left tab.

Favoriten können nicht einzeln sichtbar gemacht werden, sondern nur die gesamte Gruppe.
Deshalb ist dieses Menü in den Gruppen auf der linken Registerkarte verfügbar.

**Reply by 1maxi2** [reply]
O.k. Jetzt habe ich es begriffen.  Ist auf alle Fälle sinnvoll mit vielen Gruppen zu arbeiten. Dann spielt es auch keine Rolle wenn alle Favoriten einer Gruppe auf einmal angezeigt werden.
